src/components/MetadataPanel/style.css

```diff
@@ -0,0 +1,84 @@
+.container {
+    color: var(--dark-theme-text-color-bright);
+        padding: 16px 14px 16px 24px;
+}
+
+.collapsed {
+    padding: 14px 8px 8px 16px;
+}
+
+.agent-selected {
+    min-height: 265px;
+}
+
+.title {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 4px;
+}
+
+h3 {
+    color: var(--pure-white);
+    font-weight: 800;
+    margin: 0px;
+}
+
+.icon {
+    background-color: transparent;
+    border: none;
+    color: var(--warm-gray);
+    font-size: 18px;
+}
+
+.title:hover .icon {
+    color: var(--pure-white);
+}
+
+.rotate span {
+    transform: rotate(180deg);
+}
+
+.panel-contents {
+    color: var(--medium-gray);
+}
+
+.no-agent-row {
+    display: flex;
+    flex-direction: column;
+    color: var(--white-two);
+}
+
+.no-agent-row p {
+    margin-bottom: 4px;
+}
+
+.no-agent-text {
+    font-size: 12px;
+    font-weight: 100px;
+}
+
+.agent-missing-text {
+    font-size: 13px;
+    font-weight: 400;
+}
+
+.row {
+    display: flex;
+    font-size: 12px;
+    font-weight: 100;
+}
```

Comment:
I would put this higher, put all the default text styling here: like the display, font-size, font-weight, color, etc then I don't think you never need the `no-agent-text` class because it's the same and the only deviation is the agent-missing-text

Reply:
That's fine, but will override the styling on the text that says "Nothing to see here" so I'll need the same number of classes in the end to fix that

Reply:
Ok so I pushed some rules up to `.container` and `.panel-contents` and trimmed where I could, also switched everything to css vars.

Still needed `.no-agent-text` but moved the class onto the previous line and its just font-size now.